test/acceptance/features/steps/servicebindingoperator.py

```diff
@@ -16,7 +16,9 @@ def __init__(self,  name="service-binding-operator", namespace="openshift-operat
 
     def check_crd(self):
         crd_name = "servicebinding"
-        assert self.openshift.is_resource_in(crd_name), f"CRD '{crd_name}' does not exist"
```

Comment:
the removed line what doing exactly what the added lines are doing now. What is the motivation for the change?

Reply:
we changed the return type for the `is_resource_in` function. that's why updated this function to assert on the proper value.